candidate-standard/EDR-candidate-specification.adoc

```diff
@@ -90,14 +90,12 @@ include::clause_6_overview.adoc[]
 
 include::clause_7_core.adoc[]
 
-include::clause_8_collections.adoc[]
+include::clause_8_queries.adoc[]
 
 include::clause_9_general.adoc[]
 
 include::clause_10_oas.adoc[]
```

Comment:
This file was deleted so the include throws an error.

Reply:
Well spotted. Thought I had editted the include